DEVELOPMENT.md

```diff
@@ -529,3 +531,70 @@ $ git push origin master
 ```
 
 After pushing the tag, our CI system will start building the release. When this is done, you'll find a new release on the [releases page](https://github.com/TheThingsNetwork/lorawan-stack/releases). After this is done, you'll need to edit the release notes. We typically copy-paste these from `CHANGELOG.md`.
+
+
+### Console Troubleshooting
+
+#### Problem: Assets are not found
```

Comment:
Perhaps add newlines here to adhere with the formatting of the rest of the document. Also for the other headings